docs/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What's the reason for adding this?

Reply:
> I also excluded docs from the "no relative imports rule" and converted the imports there to be relative in order to make "docs building utils" work fine with scripts and flake/isort without having to do some "noqa: excludes"

The main reason was that isort produced different sorting order depending on was run on just "docs" or "all files". Simply sometimes "docs" were treated as implicit package in "airflow" and sometimes as standalone "implicit" package and sorting order depended on that (even adding "docs" as known package in setup.cfg did not help. converting docs to regular "standard" python package helped.